docs/metecho-how-to.md

```diff
@@ -0,0 +1,219 @@
+    a different one.
+  - Select Create to save and navigate to your Task detail view. Select Create &
+    New to continue creating multiple Tasks that are unrelated to each other.
+- If you need to edit the Task name or description, or delete the Task, select
+  the gear icon in the top right corner. Deleting a Task deletes all the Orgs as
+  well.
+
+## Step 3 - Assign Task Developer
+**Developer** - *The person assigned to do the work of a Task.*
+
+- Assign yourself or another person to be the Developer for each Task you
+  created.
+- If you do not see the person you want to assign, contact an admin for the
+  repository on GitHub and ensure that the person is a collaborator. Then
+  re-sync GitHub collaborators in Metecho.
+- If you need to change or remove the Task Developer, use the drop down menu on
+  the Developer card in the Task detail view.
+
+## Step 4 - Create Dev Org
+**Dev Org** - *A temporary Salesforce org where a Developer can work on
+contributions to a Project. Dev Orgs expire after 30 days, and all unretrieved
+work is deleted.*
+
+- Review the Next Steps for This Task list to see progress and who is
+  responsible for the next step.
+- To begin work, the Developer will need to select Create Org in the Dev Org
+  card.
+- Dev Org creation can take a number of minutes. Feel free to leave the page.
+  Metecho will provide an alert when the Dev Org is ready.
+
+## Step 5 - Make Changes in Dev Org
+- **This step takes place in the new Salesforce Dev Org, not in Metecho.**
+- Select View Org on the Dev Org card to navigate out of Metecho and make your
+  changes in the temporary Salesforce Dev Org.
+
+## Step 6 - Retrieve Changes from Dev Org
+**Retrieve Changes** - *Pull the work you did in your Salesforce Dev Org into
+Metecho so that other people can review it. Developers may retrieve changes as
+many times as they like while they are working.*
+
+- When the Developer is ready to pull the Salesforce Dev Org work into Metecho,
+  they need to select Check for Unretrieved Changes (button located just above
+  the Developer card in the Task detail view).
+- Select the location to retrieve changes.
+  - Use the preselected Package Directory unless you know of a specific reason
+    to select a different option.
+- Select the changes to retrieve (or ignore changes you do not intend to
+  retrieve).
+- Create a commit message that briefly describes the changes.
+- Select Retrieve Selected Changes.
+- Retrieving changes can take a number of minutes. Feel free to leave the page.
+  Metecho will provide an alert when the changes have been retrieved.
+- Note that the Developer, commit message, and date now appear in a Commit
+  History list in the Task detail view.
+
+## Step 7 - Submit Task Changes for Testing
+**Submit Changes** - *Document all the changes a Developer made in a Salesforce
+Dev Org so that a Tester can test the work and leave a review. This action
```

Comment:
Related to my question above, are using `Salesforce Dev Org` because these were changes made in the Salesforce UI and not Metecho?